demosplan/DemosPlanCoreBundle/ResourceTypes/StatementResourceType.php

```diff
@@ -276,6 +277,20 @@ protected function getProperties(): array|ResourceConfigBuilderInterface
 
                     return '' === $draftsListJson ? null : Json::decodeToArray($draftsListJson);
                 });
+            $configBuilder->status->readable(true, function (Statement $statement) {
+                if ($statement->isAlreadySegmented()) {
+                    $segments = $statement->getSegmentsOfStatement();
+                    foreach ($segments as $segment) {
+                        if (!$segment->getPlace()->getSolved()) {
+                            return 'processing';
+                        } else {
+                            return 'completed';
+                        }
+                    }
+                }
```

Comment:
I think it has to be like so:
```suggestion
                if ($statement->isAlreadySegmented()) {
                    $segments = $statement->getSegmentsOfStatement();
                    foreach ($segments as $segment) {
                        if (!$segment->getPlace()->getSolved()) {
                            return 'processing';
                        }
                    }
                   
                    return 'completed';
                }
```